src/components/interactive-builder/modals/question/question-form/common/concept-search/concept-search.component.tsx

```diff
@@ -23,6 +24,7 @@ const ConceptSearch: React.FC<ConceptSearchProps> = ({
   onClearSelectedConcept,
   onSelectConcept,
   retainConceptInContextAfterSearch = false,
+  clearSearchAfterSelection,
```

Comment:
Since this is an optional parameter, better to initialize this with a default value.
```suggestion
  clearSearchAfterSelection = false,
```